package.json

```diff
@@ -59,6 +59,7 @@
     "express-basic-auth": "^1.2.0",
     "fast-deep-equal": "^3.1.1",
     "glob": "^7.1.6",
+    "jsonpath": "^1.0.2",
```

Comment:
There are a bunch of dependencies in this project which aren't in the `dependencies` of `package.json`

I may come back to it before Tuesday, but I added this one for now